source/dom/dispatcher.js

```diff
@@ -54,8 +54,20 @@
 		/**
 		* @private
 		*/
+		platformSpecific: function() {
```

Comment:
The focus and blur events on the window object aren't actually specific to webOS, right? Is there any reason why we're trying to scope this change to webOS?